docs/src/Provider.doc.js

```diff
@@ -93,59 +94,71 @@ function Example(props) {
       />
     </MainSection.Subsection>
     <MainSection.Subsection
-      title="On Navigation"
-      description="This example illustrates the implementation of `onNavigation` context to control the link functionality of Gestalt components externally. This example has 4 relevant parts: a Provider, an `onNavigation` high-order function, consumer components (Link, Button, IconButton, TapArea), and `onNavigationOptions` props. The top Provider passes the `onNavigation` function to consumer components. Then, `onNavigation` returns a function that gets called during the `onClick` event handler. The `onNavigation` function can contain complex logic including React hooks to perform side effects. In this case, `onNavigation` is used to disable the link, show an alert message, and open a different URL in a new window. Finally, the `onNavigationOptions` prop provides a flexible API. In this case, `navigationMode` allows to disable/enable the `onNavigation` logic. Other uses could be accessing `event.stopPropagation`."
+      title="Custom navigation context"
+      description={`
+        These examples illustrate a custom implementation of \`onNavigation\` context to control the link functionality of Gestalt components externally.
+
+        This example has four relevant parts: a Provider, an \`onNavigation\` high-order function, consumer components and \`onNavigationOptions\` props.
+
+        The top Provider passes the custom \`onNavigation\` function to consumer components which execute it. Then, \`onNavigation\` returns a function that gets called during the \`onClick\` event handler.
+
+        The \`onNavigation\` function can contain complex logic, including React hooks, to perform side effects. It also takes named arguments: \`href\`, \`onNavigationOptions\` and \`target\`. In this case, \`onNavigation\` executes the following actions:
+          - Disable the default Link behavior
```

Comment:
One last tiny nit: "In the examples below," instead of "In this case,"

Reply:
DONE